packages/gatsby-theme-carbon/src/components/Container/Container.js

```diff
@@ -52,6 +55,7 @@ const Container = ({ children, homepage, theme }) => {
         id="main-content"
         aria-hidden={overlayVisible}
         className={containerClassNames}
+        useHeaderNavigation={navigationStyle}
```

Comment:
What is this prop doing? It seems like you might not need it, if you're already adding the styles to this same element on line 42: `'container--header--nav': navigationStyle,`

Reply:
Removed! I kept it there on accident